modules/aruco/include/opencv2/aruco/charuco.hpp

```diff
@@ -92,13 +92,14 @@ class CV_EXPORTS_W CharucoBoard : public Board {
      * @param markerLength marker side length (same unit than squareLength)
      * @param dictionary dictionary of markers indicating the type of markers.
      * The first markers in the dictionary are used to fill the white chessboard squares.
+     * @param legacy legacy chessboard pattern (pre 4.6.0)
```

Comment:
I propose to add link to the issue/pr and briefly describe the change. I know what happened and why, but it's not clear for regular developer without code digging.

Reply:
Thanks. I updated the parameter description, please check.